components/collapsible-panel/README.md

```diff
@@ -358,6 +359,36 @@ Collapsible panels have two optional slots, `header` and `actions` that can be u
 </d2l-collapsible-panel>
 ```
 
+## Group [d2l-collapsible-panel-group]
+
+In situations with multiple collapsible panels, use the `d2l-collapsible-panel-group` component to automatically handle spacing and layout.
+
+<!-- docs: demo live -->
+```html
+<script type="module">
+	import '@brightspace-ui/core/components/collapsible-panel/collapsible-panel.js';
+	import '@brightspace-ui/core/components/collapsible-panel/collapsible-panel-group.js';
+</script>
+
+<style>
+	d2l-collapsible-panel {
+		width: 800px;
+	}
+</style>
+
+<d2l-collapsible-panel-group>
+	<d2l-collapsible-panel panel-title="Submission 1">
+		<p>Sweet roll candy dessert caramels shortbread gummies toffee oat cake cookie. Wafer gummies shortbread sweet halvah jujubes sweet. Cake chocolate chocolate bar carrot cake marzipan. Icing chupa chups jujubes macaroon toffee chocolate bar wafer croissant.</p>
+	</d2l-collapsible-panel>
+	<d2l-collapsible-panel panel-title="Submission 2">
+		<p>Sweet roll candy dessert caramels shortbread gummies toffee oat cake cookie. Wafer gummies shortbread sweet halvah jujubes sweet. Cake chocolate chocolate bar carrot cake marzipan. Icing chupa chups jujubes macaroon toffee chocolate bar wafer croissant.</p>
+	</d2l-collapsible-panel>
+	<d2l-collapsible-panel panel-title="Submission 3">
+		<p>Sweet roll candy dessert caramels shortbread gummies toffee oat cake cookie. Wafer gummies shortbread sweet halvah jujubes sweet. Cake chocolate chocolate bar carrot cake marzipan. Icing chupa chups jujubes macaroon toffee chocolate bar wafer croissant.</p>
+	</d2l-collapsible-panel>
+</d2l-collapsible-panel-group>
+```
+
```

Comment:
This doesn't have official approval yet -- we can either omit it, or leave it in and iterate. 

I'm fine with either.

Reply:
I think it's ok. If Jeff wants to tweak it, we can certainly do that later.